modules/appeals_api/app/workers/appeals_api/pdf_submit_job.rb

```diff
@@ -89,20 +96,19 @@ def handle_upload_error(appeal, e)
       log_upload_error(appeal, e)
       appeal.update_status(status: 'error', code: e.code, detail: e.detail)
 
-      if e.code == 'DOC201' || e.code == 'DOC202'
-        notify(
-          {
-            'class' => self.class.name,
-            'args' => [appeal.id, appeal.class.to_s, appeal.created_at.iso8601],
-            'error_class' => e.code,
-            'error_message' => e.detail,
-            'failed_at' => Time.zone.now
-          }
-        )
-      else
-        # allow sidekiq to retry immediately
-        raise
-      end
+      # re-raise retryable EMMS errors so sidekick will retry
+      raise if RETRYABLE_EMMS_RESP_STATUS_CODES.include?(e.upstream_http_resp_status)
```

Comment:
This looks good to me. 👍 I'll wait to approve since you mentioned you're adding some unit tests.